src/Snippet/QueueProcessor.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Bolt\Snippet;
+
+use Symfony\Component\HttpFoundation\Response;
+use Tightenco\Collect\Support\Collection;
+
+/**
+ * Snippet queue processor.
+ */
+class QueueProcessor
```

Comment:
this processor and injector should be disabled by default (we can guide developers in docs to use render_widgets where they expect widgets)